npm/private/npm_translate_lock_helpers.bzl

```diff
@@ -465,13 +465,7 @@ ERROR: can not apply both `pnpm.patchedDependencies` and `npm_translate_lock(pat
             replace_package = replace_package,
         )
 
-        if repo_name in result:
```

Comment:
This is just (an escaped version of) `{name}__{package_key}` and the key should already be unique. It's not worth the extra code here...